storage/cloudspanner/spanner.sdl

```diff
@@ -0,0 +1,79 @@
+-- CloudSpanner has a strong recommendation to restrict all data between
+-- top-level table rows to less than a few GiB:
+--  | As a rule of thumb, the size of every set of related rows in a hierarchy
+--  | of parent-child tables should be less than a few GiB. A set of related
+--  | rows in a hierarchy of parent-child tables is defined as: (a single row
+--  | of a table at the root of a database hierarchy) + (all rows of that table's
+--  | descendent tables that share the row's primary key) + (all rows of
+--  | interleaved indexes that share the row's primary key).
+--  | (https://cloud.google.com/spanner/docs/schema-and-data-model)
+-- so we don't interleave our tables.
```

Comment:
This is a bit of a shame? Oh well.

Reply:
Not necessarily, we'll have a bit more coordination going on, but by the same token there'll be more parallelism to potentially gain from.   We'll have to try it and see :)